woocommerce/woocommerce-admin
# Suppress `lint:js` warnings in CI and GitHub PRs

Add returned type annotations to `packages/admin-e2e-tests/src/elements/*.ts`,
to suppress CI and GitHub warnings.


### Accessibility



-   [n/a] I've tested using only a keyboard (no mouse)
-   [n/a] I've tested using a screen reader
-   [n/a] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
-   [n/a] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

### Screenshots

#### Before 
![A list of warnings in GH PR](https://user-images.githubusercontent.com/17435/145258020-a1d96d19-312e-4fb3-b96f-7abb45925ec0.png)
```
✖ 431 problems (0 errors, 431 warnings)
```

#### After
![image](https://user-images.githubusercontent.com/17435/145275215-8872514c-4f06-4371-8ffe-2f682e698247.png)

https://github.com/woocommerce/woocommerce-admin/runs/4461433270?check_suite_focus=true#step:5:742
```
✖ 329 problems (0 errors, 329 warnings)
```


### Detailed test instructions:

1. Open the ["Files changed" tab](https://github.com/woocommerce/woocommerce-admin/pull/8020/files)
2. See fewer warnings.
3. Open [GitHub action run log](https://github.com/woocommerce/woocommerce-admin/actions/runs/1555284320),
4. See fewer lint warnings.
5. `npm run lint:js`
6. See fewer lint warnings.
